lib/postmark/api_client.rb

```diff
@@ -221,6 +221,22 @@ def deliver_with_template(attributes = {})
       end
     end
 
+    def get_stats_totals(options = {})
+      format_response(http_client.get('stats/outbound', options))
+    end
+
+    def get_stats_counts(stat, options = {})
+      url = "stats/outbound/#{stat}"
+
+      url += "/#{options.delete(:type)}" if options.has_key?(:type)
```

Comment:
Right, nice catch on the `+=`. 

On the `delete`, I was using the mutating behavior as the API doesn't support `type` in the querystring, but it's also doesn't hurt anything to pass it along as well.